include/deal.II/fe/fe_interface_values.h

```diff
@@ -307,6 +307,47 @@ class FEInterfaceValues
           const unsigned int q_point,
           const unsigned int component = 0) const;
 
+  /**
+   * Return the average of the gradient $\{\nabla u \} = \frac{1}{2}\nabla
```

Comment:
Don't we typically use double-braces for the jump?

Reply:
This is the average not the jump and I used single braces in average()

Reply:
Fair enough -- I guess I've seen it both ways.